app/src/main/java/de/danoeh/antennapod/discovery/PodcastIndexPodcastSearcher.java

```diff
@@ -0,0 +1,127 @@
+package de.danoeh.antennapod.discovery;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.security.MessageDigest;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+import java.util.TimeZone;
+
+import de.danoeh.antennapod.core.ClientConfig;
+import de.danoeh.antennapod.core.service.download.AntennapodHttpClient;
+import io.reactivex.Single;
+import io.reactivex.SingleOnSubscribe;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.schedulers.Schedulers;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+public class PodcastIndexPodcastSearcher implements PodcastSearcher {
+    private static final String PODCASTINDEX_API_URL = "https://api.podcastindex.org/api/1.0/search/byterm?q=%s";
+    private static final String API_KEY = "JRJPPWC6ZA7DKKTSU2R3";
+    private static final String API_SECRET = "7$$67JtrfkSYtAncGBEaJp$v$Y9$ZJUzYVy8GuBm";
```

Comment:
Could you please make those a `buildConfigField`? That way, we can have one key+secret that is open-source and might get blocked if someone abuses it and an additional pair that is shipped only to Google Play and is not included in the source code.

Something like this should work:
https://github.com/AntennaPod/AntennaPod/blob/develop/app/build.gradle#L32

Reply:
Changed these fields to `buildConfigField`